test/MathOptInterface/MOI_wrapper.jl

```diff
@@ -81,6 +81,45 @@ function test_runtests()
     return
 end
 
+function test_Binaryfixing()
+    @testset "Binary Equal to 1" begin
+        T = Float64
+        config = MOI.Test.Config(atol = 1e-3, rtol = 1e-3)
+        model = Xpress.Optimizer(OUTPUTLOG = 0)
+        x = MOI.add_variable(model)
+        MOI.add_constraint(model, x, MOI.EqualTo(T(1)))

```

Comment:
what happens if we do:
`MOI.add_constraint(model, x, MOI.GreaterThan(T(1//2)))`

Reply:
It works, he gives the right bound
```
Minimize
 C1

Subject To
R1: C1 >= 0.5 

Bounds

Binaries
C1 

End
```